configsnap

```diff
@@ -72,8 +76,23 @@ def create_dir(tagdir, overwrite):
     return workdir
 
 
-diffs_found_msg = False
-is_php_detected = False
+def create_archive(tagdir, workdir, tag, overwrite):
+    """Create tar archive of workdir and return filename and success"""
+    output_filename = "%s/configsnap-%s.tar.gz" % (tagdir, tag)
+
+    if os.path.isfile(output_filename):
+        if overwrite:
+            report_info(
+                "%s exists. Overwrite (-w/--overwrite) enabled so will overwrite." % output_filename)
+        else:
+            report_error(
+                "Archive file %s already exists. Use -w overwrite." % output_filename)
+            return "", False
+
+    with tarfile.open(output_filename, "w:gz") as tar:
```

Comment:
Hmm, none of this stuff is tested on EL5 any more since its unsupported. I don't really want to make the code more verbose to support it either. Given that everything else will continue to work OK, can we just not use -w on EL5, or do you think it is still worth fixing?